ssm_jax/linear_gaussian_ssm/models.py

```diff
@@ -346,11 +438,19 @@ def em_step(_params):
             _params = self.params
             return _params, marginal_loglikes.sum()
 
+        def emap_step(_params):
```

Comment:
The `emap_step` is very similar to `em_step`, maybe just merge them with an if statement to control whether to call `m_step` or `map_step`?

Reply:
make sense to me, will do